docs/versionpicker.html

```diff
@@ -0,0 +1,115 @@
+<!-- Start interactive version picker -->
+<div id="installation" v-cloak>
+  <header class="row">
+    <h3>Paste this into the <code>&lt;head&gt;</code> of your HTML:</h3>
+    <button type="button" @click="copyToClipboard">
+      <transition>
+        <span v-if="copyStatus === 'success'" class="emoji">✔</span>
+        <span v-else-if="copyStatus === 'failed'" class="emoji">❌</span>
+      </transition>
+      Copy to clipboard
+    </button>
+  </header>
+
+  <transition mode="out-in">
+    <pre :key="selectedVersion.fileSnippet"><code>{{selectedVersion.fileSnippet}}</code></pre>
+  </transition>
+
+  <h3>Version options:</h3>
+  <form action="./" method="post" @submit.prevent>
+    <label>
+      <input type="radio" value="dark" checked name="theme" v-model="versionOptions.theme" />
+      Dark theme&nbsp;<span class="emoji">🌙</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / ☀
+      </span>
+    </label>
+    <label>
+      <input type="radio" value="light" name="theme" v-model="versionOptions.theme" />
+      Light theme&nbsp;<span class="emoji">☀</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / 🌙
+      </span>
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" checked v-model="versionOptions.isStandalone" />
+      Enforce theme? (ignore <code>prefers-color-scheme</code>)
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" v-model="versionOptions.isLegacy" />
```

Comment:
Huh, strange that it shows the dark theme for you.
For me on Internet Explorer:
  - both dark and light versions work when `legacy` & `standalone` are selected
  - both dark and light versions _fail_ when `legacy`, but not `standalone` is selected.
This is expected for now, because the default legacy stylesheets are basically "proxy files", loading in either the dark or light standalone versions from jsdelivr (depending on `prefers-color-scheme`). But since these files are part of v2 and not published yet, these "chained" requests fail. (you can see that in the console)